app/models/claim/advocate_claim.rb

```diff
@@ -101,22 +101,32 @@ class AdvocateClaim < BaseClaim
             condition: ->(claim) { !claim.fixed_fee_case? }
           },
           {
-            to_stage: :basic_and_fixed_fees,
+            to_stage: :fixed_fees,
             condition: ->(claim) { claim.fixed_fee_case? }
           }
-        ]
+        ],
+        dependencies: %i[case_details]
       },
       {
         name: :offence_details,
         transitions: [
-          { to_stage: :basic_and_fixed_fees }
-        ]
+          { to_stage: :basic_fees }
+        ],
+        dependencies: %i[case_details defendants]
       },
       {
-        name: :basic_and_fixed_fees,
+        name: :basic_fees,
```

Comment:
Splitting **basic_fees** and **fixed_fees** as they should be 2 different steps anyway.